### PR TITLE
SELinux policy fixes for report scheduling/exporting/emailing

### PR DESCRIPTION
**Pull request by vpodzime**

### CODE REVIEW

**Comment by nickanderson** [comment]
impressive work here

**Comment by vpodzime** [comment]
Dealing with the notification/alert scripts is more tricky then I though so I redacted it from this PR. Let's merge this to have the report exporting and emailing working and let's have enough time to deal with notification/alert scripts properly.

File: misc/selinux/cfengine-enterprise.fc [hunk 1]
```diff
@@ -6,4 +6,4 @@
 /var/cfengine/bin/pg.*		    --	gen_context(system_u:object_r:cfengine_postgres_exec_t,s0)
 /var/cfengine/httpd/bin/.*      --	gen_context(system_u:object_r:cfengine_httpd_exec_t,s0)
 /var/cfengine/httpd/php/bin/.*  --	gen_context(system_u:object_r:cfengine_httpd_exec_t,s0)
-/opt/cfengine(/.*)?             --	gen_context(system_u:object_r:cfengine_var_lib_t,s0)
\ No newline at end of file
+/opt/cfengine(/.*)?             	gen_context(system_u:object_r:cfengine_var_lib_t,s0)
```
Comment (olehermanse) [inline misc/selinux/cfengine-enterprise.fc:9]: Is it normal that these files should not have a newline at the end?

**Review by olehermanse — APPROVED** [review]
LGTM

File: misc/selinux/cfengine-enterprise.fc (hunk 1 above)
Comment (vpodzime) [reply misc/selinux/cfengine-enterprise.fc:9]: I think it doesn't matter. So I'll make sure there's a newline at the end of the file in the follow-up work.

**Pull request merged**